vortex-dtype/src/ptype.rs

```diff
@@ -30,7 +31,6 @@ pub enum PType {
 pub trait NativePType:
     Send
     + Sync
-    + Sized
```

Comment:
Why no longer `Sized`?

Reply:
Because it's always Sized https://doc.rust-lang.org/std/marker/trait.Sized.html